### PR TITLE
[no-issue] added map widget announcement

### PR DESCRIPTION
**Pull request by janhommes**

### CODE REVIEW

File: content/release-10-16-0/announcements-10-16-0.md [hunk 1]
```diff
@@ -99,6 +99,9 @@ For example, if a user sends a request in the following format:
 
 currently the whole payload is saved. In the future, the `c8y_LatestMeasurements` fragment will be ignored and not saved.
 
+##### Breaking change in the Map widget
+With version 10.18 the Map Widget will be migrated and the support for realtime updates on all devices will be removed by an configurable refresh interval. This will allow us to display more then 100 devices on the map by default. Additionally the Map Widget will not support the "show track" option anymore.
```
Comment (BeateRixen) [inline content/release-10-16-0/announcements-10-16-0.md:103]:
```suggestion
As of release 10.18+, the "Map" widget will be migrated and support for real-time updates on all devices will be replaced by a configurable refresh interval. This change allows us to display more then 100 devices on the map by default. Additionally, the "Map" widget will no longer support the "show track" option.
```

File: content/release-10-18-0/announcements-10-18-0.md [hunk 2]
```diff
@@ -47,6 +47,9 @@ Since this release, the `c8y_LatestMeasurements` fragment will be ignored and no
 
 As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), as of release 10.18+, wrong values for `pageSize` or `currentPage` result in a 422 (Unprocessable entity) HTTP status code instead of a 500 (Internal server error) HTTP status code.
 
+##### Breaking change in the Map widget
+With version 10.18 the Map Widget will be migrated and the support for realtime updates on all devices will be removed by an configurable refresh interval. This will allow us to display more then 100 devices on the map by default. Additionally the Map Widget will not support the "show track" option anymore.
```
Comment (BeateRixen) [inline content/release-10-18-0/announcements-10-18-0.md:51]:
```suggestion
As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), with release 10.18+, the "Map" widget has been migrated and support for real-time updates on all devices has been replaced by a configurable refresh interval. This change allows us to display more then 100 devices on the map by default. Additionally, the "Map" widget no longer supports the "show track" option.
```

**Comment by BeateRixen** [comment]
@janhommes I assume that this should also go into 10.17?

**Comment by janhommes** [comment]
> @janhommes I assume that this should also go into 10.17?

yes you are right, added.

File: content/release-10-17-0/announcements-10-17-0.md [hunk 3]
```diff
@@ -62,6 +62,9 @@ This change is required to make the API consistent with the HTTP protocol semant
 
 As of release 10.18+, wrong values for `pageSize` or `currentPage` result in a 422 (Unprocessable entity) HTTP status code instead of a 500 (Internal server error) HTTP status code.
 
+##### Breaking change in the Map widget
+With release 10.18, the "Map" widget will be migrated and support for real-time updates on all devices will be replaced by a configurable refresh interval. This change allows us to display more then 100 devices on the map by default. Additionally, the "Map" widget will no longer support the "show track" option.
```
Comment (BeateRixen) [inline content/release-10-17-0/announcements-10-17-0.md:66]:
```suggestion
As announced with [release 10.16](/release-10-16-0/announcements-10-16-0), with release 10.18+, the "Map" widget will be migrated and support for real-time updates on all devices will be replaced by a configurable refresh interval. This change allows us to display more then 100 devices on the map by default. Additionally, the "Map" widget will no longer support the "show track" option.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**